docs/01-overview/main-areas/telemetry/telemetry-03-traces.md

```diff
@@ -264,6 +264,12 @@ The trace collector setup is designed using the following assumptions:
 - An unavailability of a destination must be survived for 5 minutes without direct loss of trace data
 - An average span consists of 40 attributes with 64 character length
 
+The trace collector setup is configured filter out following traces and spans:
+- `/healtz` endpoint of any component deployed on `kyma-system` namespace
+- `/metrics` endpoint of any component deployed on `kyma-system namespace
+- All traces generated from Grafana and Jaeger deployments
+- All traces in reference to `fluent-bit` and `loki` communication 
+
```

Comment:
```suggestion
System related spans reported by Istio will be filtered out without option to opt-out, especially the following:
- `/healtz` endpoint of any component deployed on `kyma-system` namespace
- `/metrics` endpoint of any component deployed on `kyma-system` namespace
- All outgoing spans reported by Grafana and Jaeger
- All spans in reference to `fluent-bit` and `loki` communication 
```

Can you please move it into a dedicated subpoint with a title like: System span filtering